javascript-bank/bank.js

```diff
@@ -9,7 +9,7 @@ Bank.prototype.openAccount = function (holder, balance) {
     return null;
   } else {
     var newAccount = new Account(this.nextAccountNumber, holder);
-    newAccount.transactions.push(new Transaction('deposit', balance));
+    newAccount.transactions.push(newAccount.deposit(balance));
```

Comment:
`newAccount.deposit()` already pushes a transaction for you. Check it out.

https://github.com/khan-osama/c0121-code-solutions/pull/104/files#diff-e5610c70925a44028775d6e7ad6a29f85559de55cdc3f49dae747fdafecb809dR8